packages/client/tests/system/CRYP-177_tags_list.e2e.ts

```diff
@@ -0,0 +1,33 @@
+import { openAppForDebugBuild } from "./utils/open_app_for_debug_build";
+import { by, device, expect, element } from "detox";
+import { pause } from "./utils/test_utils";
+
+describe("CRYP-95 Settings page", () => {
```

Comment:
Update the describe text

```suggestion
describe("CRYP-177 Tags list screen", () => {
```